app/views/partials/theme/simple/marketplace_setup/index.liquid

```diff
@@ -31,3 +31,22 @@
       <button class="mt-3 button">{{ 'app.stripe.save' | t }}</button>
     </form>
   </div>
+  </br>
+
+  <div class="mb-2 w-full md:w-full">
+    <h1 class="text-2xl">Chat</h1>
```

Comment:
Use translation

Reply:
Also, use `<h2 class="title"></h2>` and it will be styled as all the titles in the rest of the app. :) <h2> is preferred as <h1> is the title of the page. It should not matter, but as far as I know about SEO it might, so better safe than sorry.